src/pages/TeachersUnite/KnowATeacherPage.js

```diff
@@ -51,7 +53,9 @@ function KnowATeacherPage(props) {
         const firstName = values.firstName.trim();
         const lastName = values.lastName.trim();
 
-        TeachersUnite.referTeachersUniteVolunteer(contactMethod, firstName, lastName);
+        const policyID = isProduction ? CONST.TEACHERS_UNITE.PROD_POLICY_ID : CONST.TEACHERS_UNITE.TEST_POLICY_ID;
+        const publicRoomReportID = isProduction ? CONST.TEACHERS_UNITE.PROD_PUBLIC_ROOM_ID : CONST.TEACHERS_UNITE.TEST_PUBLIC_ROOM_ID;
```

Comment:
I think we can create a hook to return the teachers' unite policyID and publicRoomID based on the environment.

Reply:
This constants will be only used in these two places, so I think it's not necessary to move it to a hook atm